analytics.js

```diff
@@ -194,6 +208,12 @@ class Analytics {
         );
       } catch (e) {
         logger.error(e);
+        if (window.newBugsnag) {
+          window.newBugsnag.leaveBreadcrumb(
+            `cookieConsent initialization failed`
+          );
+          window.newBugsnag.notify(e);
+        }
```

Comment:
It'd be difficult to always check for the Bugsnag client and notify the error, increasing the code footprint.

Reply:
As we are loading and initializing Bugsnag asynchronously it is hard to tell when it is finally done with the initialization.

Reply:
We need to refactor this to avoid repeating it.

Reply:
Done